pandas/core/frame.py

```diff
@@ -1969,6 +1973,38 @@ def memory_usage(self, index=True, deep=False):
         See Also
         --------
         numpy.ndarray.nbytes
+
+        Examples
+        --------
+        >>> dtypes = ['int64', 'float64', 'complex128', 'object', 'bool']
+        >>> data = dict([(t, np.random.randint(100, size=5000).astype(t))
+        ...              for t in dtypes])
+        >>> df = pd.DataFrame(data)
+        >>> df.memory_usage()
+        Index            80
+        int64         40000
+        float64       40000
+        complex128    80000
+        object        40000
+        bool           5000
+        dtype: int64
+        >>> df.memory_usage(index=False)
```

Comment:
I'm not certain the two latter examples (with `index=False`) bring anything. Just the first example might be enough.